dd-sdk-android/src/main/java/com/datadog/opentracing/DDTracer.java

```diff
@@ -122,9 +122,9 @@ protected DDTracer(final Config config, final Writer writer, final Random random
                 config.getServiceName(),
                 writer,
                 Sampler.Builder.forConfig(config),
-                HttpCodec.createInjector(Config.get()),
-                HttpCodec.createExtractor(Config.get(), config.getHeaderTags()),
-                new ContextualScopeManager(Config.get().getScopeDepthLimit(), createScopeEventFactory()),
+                HttpCodec.createInjector(config),
+                HttpCodec.createExtractor(config, config.getHeaderTags()),
+                new ContextualScopeManager(config.get().getScopeDepthLimit(), createScopeEventFactory()),
```

Comment:
In theory yes, but for the sake of this PR it doesn't make a difference because we don't alter the properties that the `ContextualScopeManager` accesses. 
I think we should avoid changing the original lib as much as possible.